src/commands/doctor.ts

```diff
@@ -45,13 +54,23 @@ export default class Doctor extends SfdxCommand {
   private outputDir: string;
   private filesWrittenMsgs: string[] = [];
 
+  /**
+   * Only made into its own method for unit testing purposes
+   *
+   * @param url: url string to open
+   */
+  public static async openUrl(url: string): Promise<void> {
+    await open(url);
+  }
```

Comment:
What do you think about making this a private method and stubbing it in the test?  I know you'll have to add a `ts-ignore` and an eslint-ignore but I think that's fine to have in a test.